modules/ossm-mt-known-issues.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/installing-mt-ossm.adoc
+
+[id="ossm-mt-known-issues_{context}"]
+= Known issues with multi-tenant {ProductName} installations
+
+[WARNING]
+====
+Automatic route creation is currently incompatible with multi-tenant {ProductShortName} installations. Ensure that it is disabled, by setting `ior_enabled` to `false` in your `ServiceMeshControlPlane` if you plan to attempt a multi-tenant installation.
+====
+
+== Known issues
```

Comment:
Add an explicit anchor.

Reply:
I just removed this heading. It seemed redundant when I read through it again.